src/nxlink.c

```diff
@@ -513,12 +513,9 @@ int main(int argc, char **argv) {
 		return -1;
 	}
 
-
-	char *msyscon = getenv("MSYSCON");
-
-	if (msyscon && 0 == strcmp(msyscon,"mintty.exe")) {
-		setvbuf(stdout, 0, _IONBF, 0);
-	}
+#ifdef _WIN32
```

Comment:
Use same alignment as nearby code?

Reply:
traditionally preprocessor directives start in first column because once upon a time compilers didn't parse them if they weren't there. I guess it doesn't matter that much any more but all the directives in this file are done that way so maybe just leave it this way for now?